plots/laws/thermodynamics/quantum_statistics/fermi_dirac_temperature_dependence.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+from sympy import symbols, Eq, solve, S
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import print_expression, units
+from symplyphysics.laws.thermodynamics.fermi_dirac_statistics import (
+    single_particle_state_distribution as distribution_law,
+)
+
+occupancy = distribution_law.occupancy_of_state
+energy = distribution_law.energy_of_state
+chemical_potential = distribution_law.total_chemical_potential
+temperature = distribution_law.temperature
+
+reduced_temperature = symbols("reduced_temperature")
+reduced_temperature_eqn = Eq(
+    reduced_temperature,
+    units.boltzmann_constant * temperature / (energy - chemical_potential),
```

Comment:
Law for this?

Reply:
I don't think that's needed. This is only used once for the plot

Reply:
Well it looks rather interesting for a separate law. It is a parameter of plot but I do not see where it comes from.